extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -156,45 +162,71 @@ public function getMonetizationUnits( $params ) {
 	}
 
 	/**
+	 * Process data for the ad units
```

Comment:
Maybe adding a bit more context of why we are processing the data would be helpful in the future.